cves/CVE-2014-9604.yml

```diff
@@ -90,16 +93,18 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 3881606240953b9275a247a1c98a567f3c44890f
-  note: ''
-- commit: ''
-  note: ''
+  note: |
+    This was the only commit made for the fix, and it just added an if statement to stop
+    the process if the slice height was zero.
+    Found-by: Mateusz "j00ru" Jurczyk and Gynvael Coldwind
+    Signed-off-by: Michael Niedermayer <michaelni@gmx.at>
 vccs:
-- commit: b5c3f0b9940f964cdcaf33907eee91f9826c00b8
-  note: Identified by archeogit,
 - commit: 0d8506b8c52659c5bfff9535391d5e95ddcd45f1
-  note: Identified by archeogit,
+  note: |
+    Commited by Kostya Shishkov, first introduced the vulnerability in the restore_median function
```

Comment:
I would like to see that this commit was discovered by a tool, please restore that portion.